src/lib/pickers/Model.svelte

```diff
@@ -94,7 +92,7 @@
                     // Found Model
                     storeFS(connection.result);
                 } else {
-                    open = true;
+                    popup.classList.add("modal-open");
```

Comment:
Can you write a check, if there's an undefined value from the file sizes to not show the Modal? Currently what happens is, it creates a Modal without the user performing any selection

If you think of a better way to do this, I'd love to hear your thoughts ^_^

Reply:
Okay so the thing is we don't have the size variable yet here, but we can work with the selected model instead right so if there is none, That will be a fine approach so i just did that if there is no selected model it doesn't So i just used that instead also i think there might be something to do but that is something we can discuss later ig once we get it working normally <3